crates/cli/src/args.rs

```diff
@@ -260,9 +261,26 @@ pub struct Args {
     /// Event signature for log decoding
     #[arg(long, value_name = "tracer", help_heading = "Dataset-specific Options")]
     pub js_tracer: Option<String>,
+    /// Keep raw decoded columns like topic0, topic1, etc.
+    #[arg(long, help_heading = "Output Options")]
+    pub keep_raw_decoded: bool,
 }
 
 impl Args {
+    /// Converts each function signature in `function` to its corresponding Ethereum function selector
```

Comment:
this should only convert functions that are not in hex form

that way someone cal specify as either `--function totalSupply()` or `--function 0xd74d829e`